templates/tensorrt-6.0-cuda10.0/src/__ALGO__.py

```diff
@@ -0,0 +1,62 @@
+import Algorithmia
+import numpy as np
+import os
+from .auxillary import load, allocate_buffers, do_inference
+from PIL import Image
+# API calls will begin at the apply() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+
+SIMD_ALGO = "util/SmartImageDownloader/0.2.14"
+MODEL_PATH = "data://algorithmiahq/collection/mobilenetv2-1.0.onnx"
```

Comment:
Is this file publicly accessible?

Reply:
it will be, good catch moving it there now

Reply:
its public now